float32[6] pos_head
float32[6] pos_left_hand
float32[6] pos_right_hand